pkg/nodeconfig/payload/payload_test.go

```diff
@@ -74,7 +74,7 @@ $cni_template=$cni_template.Replace("provider_address",$provider_address)
 # Compare CNI config with existing file, and replace if necessary
 $existing_config=""
 if(Test-Path -Path c:\k\cni.conf) {
-    $existing_config= Get-Content -Path "c:\k\cni.conf"
+` + "    $existing_config=((Get-Content -Path \"c:\\k\\cni.conf\" -Raw) -Replace \"`r\",\"\")" + `
```

Comment:
Does it make sense to test with a cni_template that contains the carriage returns so that the proposed sustitution can be unit tested?

Reply:
At the moment there is no way I can think of to unit test this script.
Our testing framework isn't set up to do individual tests like this on Windows nodes, and certainly not for parts of the powershell scripts.

Reply:
As far as confirming this works, outside of my manual testing, I'd like to see a decrease in the number of `failed to reload cni configuration after receiving fs change event` in e2e containerd logs.
[Previous run before this PR](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/1949/pull-ci-openshift-windows-machine-config-operator-master-aws-e2e-operator/1734746775840886784/artifacts/aws-e2e-operator/windows-e2e-operator-test/artifacts/nodes/ip-10-0-28-68.us-west-1.compute.internal/containerd/containerd.log)

Reply:
I've confirmed from the logs that after the initial setup, there are no more containerd errors reloading the cni.conf file.